build_bins.sh

```diff
@@ -24,23 +24,23 @@ rm -f /tmp/build_bins_chef_config.?????????.rb
 #
 DIR=`dirname $0`
 mkdir -p $DIR/bins
-pushd $DIR/bins/
+pushd $DIR/bins/ > /dev/null
 
 if [ $(dpkg-query -W -f='${Status}' chefdk 2>/dev/null | grep -c 'ok installed') -eq 0 ]; then
```

Comment:
I am not worried about counting needlessly.   This is a reliable comparison that works correctly on all known dpkg-query output.